tests/disaster-recovery/mdr/test_application_failover_and_relocate.py

```diff
@@ -0,0 +1,216 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import mdr_test, polarion_id
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.ocs.node import (
+    wait_for_nodes_status,
+    get_node_objs,
+)
+from ocs_ci.helpers.dr_helpers import (
+    enable_fence,
+    enable_unfence,
+    get_fence_state,
+    failover,
+    relocate,
+    set_current_primary_cluster_context,
+    set_current_secondary_cluster_context,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    wait_for_all_resources_creation,
+    wait_for_all_resources_deletion,
+    validate_data_integrity,
+    gracefully_reboot_nodes,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@mdr_test
+class TestApplicationFailoverAndRelocate(ManageTest):
+    """
+    Test Failover and Relocate actions for a busybox application
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, rdr_workload):
```

Comment:
Maybe we would like to rename this argument to `dr_workload`.

Reply:
As mentioned above I'll take this in enhancement PR, as we need to change in RDR tests too